frontend/src/components/common/modal/Modal.tsx

```diff
@@ -6,9 +7,11 @@ import styles from './Modal.module.scss'
 const Modal = ({
   onClose,
   children,
+  contentClassName,
```

Comment:
This doesn't actually seem to be used?
And shouldnt the styles on the content be applied on the children directly?
eg 
```tsx
modal.setModalContent(<CreateModal className={contentClassName} name={selectedName} channelType={selectedChannel} />)

// CreateModal
const CreateModal = ({className, name, channelType}...)=> {

return (<div className={cx(styles.section, className)}>...</div>)
}
```

Reply:
Yeah this one's kinda hacky. I had to add this because the CreateModal is the only modal that uses a non-standard padding/width. The issues lies in where the Modal's padding is applied:

```jsx
<div
   id={modalBackgroundId}
   className={styles.modalBg}
   onClick={handleClickBackground}
>
   <div className={styles.modal}>
      <CloseButton onClick={onClose} className={styles.close} />
      <div className={cx(styles.content, contentClassName)}>{children}</div>
   </div>
</div>
```

The padding is applied on the `div` wrapping `{children}`. So applying the padding to the children can only increase the default padding (by adding more padding to the children) but not decrease it, which is what we need. 

Alternative is to get rid of the `div` wrapping the content and have each Modal implement their own padding. But felt that was too big of a change for one exception case though.